src/player.cpp

```diff
@@ -3980,9 +3963,9 @@ bool Player::removeOutfit(uint16_t lookType)
 
 bool Player::removeOutfitAddon(uint16_t lookType, uint8_t addons)
 {
-	for (OutfitEntry& outfitEntry : outfits) {
-		if (outfitEntry.lookType == lookType) {
-			outfitEntry.addons &= ~addons;
+	for (auto& outfit : outfits) {
+		if (outfit.first == lookType) {
+			outfit.second &= ~addons;
```

Comment:
Tip for this recurring pattern:

```suggestion
	for (auto& [first, second] : outfits) {
		if (first == lookType) {
			second &= ~addons;
```

(and pick better names :sweat_smile: )

Reply:
wasn't aware that you could do it like this, will change that

Reply:
> wasn't aware that you could do it like this, will change that

You can also name them like normal variables, so for example something like `for (auto& [playerLookType, playerAddons] : outfits) {` could be even clearer

I think that method was introduced in C++14 or C++17, but I could be wrong

Reply:
thanks, it's solved in https://github.com/otland/forgottenserver/pull/4197/commits/83573ead045b1fecca79a482cc4c87197d3f9b6d